apps/judicial-system/api/src/app/modules/case/case.service.ts

```diff
@@ -65,7 +69,52 @@ export class CaseService {
     return []
   }
 
-  async sendNotificationByCaseId(id: string): Promise<Notification> {
-    return new Notification()
+  async sendNotificationByCaseId(existingCase: Case): Promise<Notification> {
+    this.logger.debug(
+      `Sending a notification for case with id "${existingCase.id}"`,
+    )
+
+    const smsText = this.constructSmsText(existingCase)
+
+    this.logger.debug(smsText)
+
+    // Production or local development with judge phone number
+    if (environment.production || environment.notifications.judgePhoneNumber) {
+      await this.smsService.sendSms('8589030', smsText)
```

Comment:
Seems kind of fishy having the phonenr visible like this in a public repo. Maybe we should store the phone number as a secret in AWS?

Reply:
Þetta er reyndar bara mitt símanúmer en vel spottað. Átti ekki að vera harðkóðað.

Fixed